mne/channels/channels.py

```diff
@@ -98,7 +98,7 @@ def _get_ch_type(inst, ch_type):
     then grads, then ... to plot.
     """
     if ch_type is None:
-        for type_ in ['mag', 'grad', 'planar1', 'planar2', 'eeg']:
+        for type_ in ['mag', 'grad', 'planar1', 'planar2', 'eeg', 'ref_meg']:
```

Comment:
same here -- have to be careful about changing these functions, recommend adding `allow_ref_meg=False` default and passing `allow_ref_meg=ref_meg` in ICA uses